docfx_yaml/extension.py

```diff
@@ -252,13 +256,21 @@ def _resolve_reference_in_module_summary(pattern, lines):
                 # Find the last component of the target. "~Queue.get" only returns <xref:get>
                 ref_name = ref_name[index:]
 
-            else:
+            elif pattern == REF_PATTERN_LAST:
                 index = matched_str.rfind('.') + 1
                 if index == 0:
                     # If there is no dot, push index to not include tilde
                     index = 1
                 ref_name = matched_str[index:]
 
+            elif pattern == REF_PATTERN_BRACKETS:
+                lbracket = matched_str.find('[')+1
+                rbracket = matched_str.find(']')
+                ref_name = matched_str[lbracket:rbracket]
+
+            else:
+                raise ValueError('Encountered wrong ref pattern.')
```

Comment:
Include the ref pattern in the exception message.

Reply:
Done.